.github/workflows/rust.yml

```diff
@@ -35,6 +35,13 @@ jobs:
       - name: Remove pre-generated prost files to force regeneration
         run: rm proto/*.rs
 
+      - name: Set precise version to allow building on 1.64.0
```

Comment:
How about commit `Cargo.lock` into the repo so that developers don't need to figure all of those?

Reply:
Nice suggestion. I've read https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html and https://github.com/rust-lang/cargo/commit/3bb02f23a23f90261b44a107e1b8821a504bac08. It seems good to just commit the `.lock` file 🍻.